.travis.yml

```diff
@@ -15,6 +15,7 @@ before_install:
   - go get -v github.com/golang/lint/golint
   - go get github.com/mattn/goveralls
   - sudo ./scripts/build-vpp.sh
+  - sudo apt-get update
```

Comment:
It should probably be before running build-vpp.sh script, since that's where some package incompatibility issues arise.